### PR TITLE
Tooling doc updates

### PR DESCRIPTION
**Pull request by cstirdivant**
Typo fixes and minor edits

### CODE REVIEW

File: tools/validators/instance_validator/README.md [hunk 1]
```diff
@@ -90,40 +96,40 @@ results in these actions:
...
-          present_value: "temp_1"
-          units:
-            key: "units"
-            values:
-              degrees_celsius: "degC"
-              degrees_fahrenheit: "degF"
-
-#### Warnings
+The following entity block would throw an error because it does not contain a type or a code.
+```yaml
```
Comment (trav3711) [inline tools/validators/instance_validator/README.md:107]: nit: remove "yaml"

File: tools/validators/instance_validator/README.md [hunk 2]
```diff
@@ -90,40 +96,40 @@ results in these actions:
...
-      links:
-        547a33c5-0ce4-4dfb-9924-5169a3475c89:
-          supply_air_flowrate_sensor: supply_air_flowrate_sensor_2
-        2b152e08-3440-4358-9067-3f2acc58c982:
-          zone_air_temperature_sensor: zone_air_temperature_sensor
-          zone_air_temperature_setpoint: zone_air_temperature_setpoint
-      type: HVAC/VAV_SD_CSP
-
+The following entity block would only expose a warning because these are not the correct fields for VAV_SD_DSP as defined in DBO.
+```yaml
```
Comment (trav3711) [inline tools/validators/instance_validator/README.md:122]: nit: remove "yaml"

**Review by trav3711 — APPROVED** [review]
two small nits.

File: tools/validators/instance_validator/README.md (hunk 1 above)
Comment (cstirdivant) [reply tools/validators/instance_validator/README.md:107]: including `yaml` formats the code block as if it were yaml text

**Pull request merged**